src/components/aboutUs/TeamDetail.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+
+//MUI styles
+import Typography from "@mui/material/Typography";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Avatar from "@mui/material/Avatar";
+import Box from "@mui/material/Box";
+
+// Defined an interface properites
+export interface TeamDetailProps {
+    teamDetail: {
+        avatar: string;
+        username: string;
+        position: string;
+        describe: string;
```

Comment:
'description' would be a better key than describe.
Also, it may be better to use our actual names rather than a username

Reply:
Thanks for the suggestions, it has been revised.